docker-compose.yml

```diff
@@ -42,7 +42,7 @@ services:
     ports:
       - "5532:5432"
     volumes:
-      - /srv/django-socio-grpc/database/django-socio-grpc:/var/lib/postgresql/data
+      - django-socio-grpc-data:/var/lib/postgresql/data
```

Comment:
windows or mac user ? 
I usually used file system to avoid loosing my data after a docker prune but I guess the best practices are more important than my personnal usage.

Reply:
Mac, but it was changed because it was hard coded path which doesn't work on Mac.

Reply:
Yes this is not working on mac and windows. Mac need a special folder sharing.